src/findings_data_import.py

```diff
@@ -114,16 +394,12 @@ def import_data(
     successful.
 
     """
-    # Boto3 clients for S3 and SSM
+    # Boto3 client for S3
     s3_client = boto3_client("s3")
-    ssm_client = boto3_client("ssm")
-
-    # Securely create a temporary file to store the JSON data in
-    temp_file_descriptor, temp_data_filepath = tempfile.mkstemp()
-
+    temp_data_filepath = None
     try:
         # This allows us to access keys with spaces in them. When they are passed
-        # in to the lambda the psaces are replaced with plus signs which results
+        # in to the lambda the spaces are replaced with plus signs which results
```

Comment:
Since we're referring to the AWS Lambda product.

```suggestion
        # in to the Lambda the spaces are replaced with plus signs which results
```